R/pkg/inst/worker/daemon.R

```diff
@@ -18,10 +18,11 @@
 # Worker daemon
 
 rLibDir <- Sys.getenv("SPARKR_RLIBDIR")
-script <- paste(rLibDir, "SparkR/worker/worker.R", sep = "/")
+dirs <- strsplit(rLibDir, ",")[[1]]
+script <- paste(dirs[[1]], "SparkR/worker/worker.R", sep = "/")
```

Comment:
this is about the result of strsplit, which is a little bit strange. for example
```> strsplit("a,b,c", ",")
[[1]]
[1] "a" "b" "c"
```

For File.separator, will fix it